fitnesse/resources/bootstrap-plus/templates/skeleton.vm

```diff
@@ -80,10 +80,41 @@
    #parse( $headerTemplate )
   </header>
 
+
   <article>
+   #if(!$bodyClass.equals("edit"))
+    ## Page Header
+    <a href="?testHistory">
+     <span class="btn btn-primary">Test History</span>
+    </a>
+    <a href="?pageHistory">
+     <span class="btn btn-primary">Page History</span>
+    </a>
+    <a href="?properties">
+     <span class="btn btn-secondary">Properties</span>
+    </a>
+    <br>
+    <hr>
+
+    #if($pageTitle.getTitle().equalsIgnoreCase("FrontPage"))
+     ## Tooltips
+     <div id="tooltip-div">
+      <p id="tooltip-text"></p>
+     </div>
+
+    #end
+   #end
+
    #parse( $mainTemplate )
   </article>
 
+  ## Tooltips edit page
+  #if($bodyClass.equals("edit"))
```

Comment:
de body class van een editor-scherm is editPage. Nu komen er geen tooltips op de edit pagina